R/write_codemeta.R

```diff
@@ -74,4 +79,16 @@ write_codemeta <- function(
   # Create or update codemeta and save to disk
   create_codemeta(pkg = pkg, root = root, use_filesize = use_filesize) %>%
     jsonlite::write_json(path, pretty = TRUE, auto_unbox = TRUE, ...)
+
+  # Create minimeta and save to disk
+  if (write_minimeta) {
+    if (!requireNamespace("jsonld", quietly = TRUE)) {
+      stop("Package jsonld required. Please install before re-trying.")
+    }
+    schemaorg <- system.file("schema", "schemaorg.json",
+                             package="codemetar")
+   jsonld::jsonld_frame("codemeta.json", schemaorg) %>%
```

Comment:
Okay cool, all is well.  Maybe slightly prettier though if we just add a `%>% jsonld::compact('{"@context": "http://schema.org"}'` in here after the frame. (not tested)!